deploy/eks/action.yml

```diff
@@ -17,11 +17,34 @@ inputs:
   AWS_REGION:
     description: AWS region to deploy to stored as secret
     required: true
-
+  SLACK_SUCCESS_CHANNEL_ID:
+    description: The Slack channel ID(s) to send successful deployment notifications
+    required: false
+  SLACK_FAILURE_CHANNEL_ID:
+    description: The Slack channel ID(s) to send failed deployment notifications
+    required: false
+  SLACK_BOT_TOKEN:
+    description: The Slack bot token to pass the data to
+    required: false
 
 runs:
-  using: 'composite'
+  using: "composite"
   steps:
+    - name: Set branch variable
+      id: set-branch
+      shell: bash
+      run: |
+        if [ "${{ github.event_name }}" == "pull_request" ]; then
+          branch="${{ github.event.pull_request.head.ref }}"
+        else
+          branch="${{ github.ref }}"
+          branch="${branch#refs/heads/}"
+        fi
+        echo "branch=${branch}" >> $GITHUB_OUTPUT
+    - name: Get short sha
+      id: get-short-sha
+      run: echo "short_sha=`echo ${GITHUB_SHA::7}`" > $GITHUB_OUTPUT
+      shell: bash
```

Comment:
Should we make this its own action, since we repeat it in every action here?
I can't recall if `$GITHUB_OUTPUT` is shared between composite actions, might be another one of those weird composite action uses composite action cases.

Reply:
Agreed, was on my "later" list. I'll give it a try.

Reply:
Going to kick the can on this too. I think we should, but higher value things to fix for now (like SemVer, and getting this rolled out further).